src/pages/exception/403/locales/zh-CN.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  'exceptionand403.exception.back': '返回首页',
+  'exceptionand403.description.403': '抱歉，你无权访问该页面。',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'exceptionand403.description.403' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)